client/app/dedicatedCloud/security/kms/add/dedicatedCloud-security-kms-add.html

```diff
@@ -0,0 +1,116 @@
+<div data-wizard
```

Comment:
Wouldn't a `oui-modale` work too ?

Reply:
There is no `oui-modal` with steps.
(But it is a CX issue, and since they didn't made any prototype, I'm stick with a modal stepper so...)

Reply:
A `oui-modal` with a `oui-stepper` then maybe, but I think this kind of stuff should be in a full screen. I know you didn't get a mock-up and that you did the best with what you add.